doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
```

Comment:
I don't think showing a 5-10 line nodejs code in the docs will impact how people understand the system. Putting two reverse proxies in place will make people ask the question: why do I need istio proxy when I already have Nginx proxy.

Publishing a simple image is not much of an issue as we have the source for the image available as well.